front-end/src/app/page.tsx

```diff
@@ -1,6 +1,6 @@
 import { LoginForm } from "@/components/Auth/LoginForm";
 
-export default function Home() {
+export default async function Home() {
```

Comment:
Is er een reden voor waarom deze async is nu?

Reply:
Nope goed gezien